### ISSUE TITLE
10.15.5 issue Internet Pane blank

### ISSUE THREAD

**Issue by dvsjr** [Lord-Kamina/SwiftDefaultApps#33]
Version SwiftDefaultApps, v. 2.0.1 build 334
Internet Pane top left when clicked shows blank pages. (See screenshot)
![SwiftDefaultApps, v  2 0 1 build 334](https://user-images.githubusercontent.com/2735427/84272321-433ff680-aafb-11ea-960e-25e246a16e90.png)

**Comment by tobsen** [comment]
10.15.6 seems to fix it.

**Comment by Cueball** [comment]
10.15.7 and it's still present.

![image](https://user-images.githubusercontent.com/7557/103663957-a8ce6580-4fac-11eb-845a-1ef7ec66145c.png)

However: it *was* working a couple of days ago. I'll check again after a reboot (but I tend to go as long as possible without reboots).

**Comment by dvsjr** [comment]
> 10.15.7 and it's still present.
> 
> ![image](https://user-images.githubusercontent.com/7557/103663957-a8ce6580-4fac-11eb-845a-1ef7ec66145c.png)
> 
> However: it _was_ working a couple of days ago. I'll check again after a reboot (but I tend to go as long as possible without reboots).

Is it only Internet that is blank? I checked and Im on 10.15.7 (19H505) and the internet app tab is blank, but the other panes are not.

First tab Internet is blank: 
![Screen Shot 2021-01-07 at 4 16 53 PM](https://user-images.githubusercontent.com/2735427/103946009-1214bc80-5104-11eb-97c6-2d6d2763b0e4.png)

second pane URI *not* blank
![Screen Shot 2021-01-07 at 4 19 30 PM](https://user-images.githubusercontent.com/2735427/103946055-248ef600-5104-11eb-878b-6b5ab5b9a5b2.png)

**Comment by dvsjr** [comment]
Is there any steps to try troubleshooting?

**Comment by Cueball** [comment]
Yes, just the Internet pane [I thought the issue title was enough, but I see how it could be ambiguous].
